File: PHP/Ruster exercise/Ruster/Deductible.php
<?php
include_once 'ARL.php';
include_once 'Health.php';
include_once 'Pension.php';

class Deductible {
    private float $calculateDeductible;

    public function __construct(float $salary) {
        $arl = new ARL($salary);
        $health = new Health($salary);
        $pension = new Pension($salary);

        $this->calculateDeductible = $arl->getCalculateARL() + $health->getCalculateHealth() + $pension->getCalculatePension();
    }

    public function getCalculateDeductible(): float {
        return $this->calculateDeductible;
    }
}

File: JAVA/Ruster exercise/com/Ruster/Deductible.java

/**
 *
 * @author JUAN
 */
package com.Ruster;

public class Deductible {
    private double calculateDeductible;
    private ARL arl;
    private Health health;
    private Pension pension;
    
    public Deductible(double salary){
        arl= new ARL(salary);
        health= new Health(salary);
        pension= new Pension(salary);
        
        this.calculateDeductible= arl.getCalculateARL() + health.getCalculateHealth() + pension.getCalculatePension();
    }
    public double getCalculateDeductible(){
        return this.calculateDeductible;
    }



}
